Runtime/MeshOperations/CombineMeshes.cs

```diff
@@ -18,7 +18,105 @@ public static class CombineMeshes
         /// A list of merged meshes. In most cases this will be a single mesh. However it can be multiple in cases
         /// where the resulting vertex count exceeds the maximum allowable value.
         /// </returns>
+        [Obsolete("Combine(IEnumerable<ProBuilderMesh> meshes) is deprecated. Plase use Combine(IEnumerable<ProBuilderMesh> meshes, ProBuilderMesh meshTarget).")]
```

Comment:
Why not keep both Combine(*)?

Reply:
@thomas-tu  this was done based on suggestion from @karl-  https://github.com/Unity-Technologies/com.unity.probuilder/pull/190#discussion_r313939477

Reply:
Oh okey, I missed it, thanks!